qemu/tests/cfg/numa_opts.cfg

```diff
@@ -101,11 +101,17 @@
             numa_expected = "0-63"
         - nodes.128:
             no aarch64
+            x86_64:
+                no RHEL.9.2
             type = numa_maxnodes
             numa_nodes = 128
-            mem_fixed = 4G
-            vm_mem_minimum = 4G
-            node_size = 32M
+            mem_fixed = 16G
+            vm_mem_minimum = 16G
+            node_size = 128M
+            RHEL.7, RHEL.8, RHEL.9.0, RHEL.9.1:
+                mem_fixed = 4G
+                vm_mem_minimum = 4G
+                node_size = 32M
```

Comment:
May be use the same configuration for all versions ?

Reply:
@yanan-fu 
Except from RHEL 9.3 onwards that same one may be used, is there a different way to set it?

Reply:
I mean,  could we use the new configuration for the old version together ?

Because, looks no defect to use `node_size = 128M` for the old product version.

Reply:
Oh sure! Code updated :grinning: